### DISCUSSION TITLE
Put breakpoint on last line in command line application

### DISCUSSION THREAD

**Discussion by liamlim** [dotnet/runtime#61358] [Q&A]
How is this not a bug? How exactly is a developer supposed to debug last line in applications like that? The reasoning makes no sense to me. You can actually set breakpoints to line without a code, you can set breakpoint on closing curly brace } for being able to debug last instruction of any method. How is this even different?

https://developercommunity.visualstudio.com/t/Cannot-set-breakpoint-on-last-line-Comma/1573483

**Comment by huoyaoyuan** [comment]
I'm not sure where this issue belongs to, but probably not runtime.
